tests/e2e/interOp/iOS/client_connectivity/test_NatMode.py

```diff
@@ -0,0 +1,126 @@
+from logging import exception
+import unittest
+import warnings
+from perfecto.test import TestResultFactory
+import pytest
+import sys
+import time
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.common.by import By
+from appium import webdriver
+from selenium.common.exceptions import NoSuchElementException
+
+import sys
+#pytestmark = [pytest.mark.ClientConnectivity, pytest.mark.nat, pytest.mark.interop]
+
+if 'perfecto_libs' not in sys.path:
+    sys.path.append(f'../libs/perfecto_libs')
+
+from iOS_lib import closeApp, openApp, verifyUploadDownloadSpeediOS, verifyUploadDownloadSpeediOSRemoteDriver, ForgetWifiConnection, Toggle_AirplaneMode_iOS, set_APconnMobileDevice_iOS, verify_APconnMobileDevice_iOS, Toggle_WifiMode_iOS, tearDown
+
+@pytest.mark.ClientConnectivity
+@pytest.mark.wifi5
+@pytest.mark.wifi6
+@pytest.mark.parametrize(
+    'setup_profiles, create_profiles',
+    [(["NAT"], ["NAT"])],
+    indirect=True,
+    scope="class"
+)
+
+@pytest.mark.usefixtures("setup_profiles")
+@pytest.mark.usefixtures("create_profiles")
+class TestNatMode(object):
+
+    @pytest.mark.fiveg
+    @pytest.mark.wpa2_personal
+    def test_ClientConnectivity_5g_WPA2_Personal(self, setup_profile_data, get_APToMobileDevice_data, setup_perfectoMobileWeb):
+        
+        profile_data = setup_profile_data["NAT"]["WPA2_P"]["5G"]  
+        ssidName = profile_data["ssid_name"]
+        ssidPassword = profile_data["security_key"]
+        print ("SSID_NAME: " + ssidName)
+        print ("SSID_PASS: " + ssidPassword)
+
+        report = setup_perfectoMobileWeb[1]
+        driver = setup_perfectoMobileWeb[0]
+        connData = get_APToMobileDevice_data
+
+        #Set Wifi/AP Mode
+        #set_APconnMobileDevice_iOS(ssidName, ssidPassword, setup_perfectoMobile_iOS, connData)
+
+        #Verify Upload download Speed from device Selection
+        verifyUploadDownloadSpeediOS(setup_perfectoMobileWeb, connData)
+        #verifyUploadDownloadSpeediOSRemoteDriver(setup_perfectoMobileWeb, connData)
+
```

Comment:
Assert statement is missing here

Reply:
fixed updated, it was a test fixture for internal testing. and its removed.